tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Coverage.java

```diff
@@ -66,13 +66,19 @@ static Coverage getOnlyTheseSources(Coverage coverage, Set<String> sourcesToKeep
     }
     Coverage finalCoverage = new Coverage();
     for (SourceFileCoverage source : coverage.getAllSourceFiles()) {
-      if (sourcesToKeep.contains(source.sourceFileName())) {
+      if (!isCcSourceFile(source.sourceFileName())
+          || sourcesToKeep.contains(source.sourceFileName())) {
         finalCoverage.add(source);
       }
     }
     return finalCoverage;
   }
 
+  private static boolean isCcSourceFile(String filename) {
```

Comment:
Can you use CppFileTypes.C_SOURCE + CPP_SOURCE?

Reply:
That means packing an additional dependency into bazel_tools which makes the binary bigger. I copied those values to Constants.CC_EXTENSIONS and use it here to check if the file is a cc file.